spec/paperclip/media_type_spoof_detector_spec.rb

```diff
@@ -43,4 +43,14 @@
       Paperclip.options[:content_type_mappings] = {}
     end
   end
+
+  it 'doest allow array as :content_type_mappings' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.